src/start_server.ts

```diff
@@ -100,11 +100,17 @@ export function getApp(options: ServerOptions) {
 
   app.use('/components/', polyserve);
 
+  const isFileRequest = (filePath: String) => {
+    return ['html', 'js', 'json', 'css', 'png', 'jpg', 'jpeg', 'gif'].filter((ext) => {
```

Comment:
Yes I was thinking of regex match, but hesitated because of the "e-mail regex" problem. We might catch too many, too few, what is the length of the file extensions etc...

A basic one would be `filePath.matches(/.+\.\w+$/)`. This could potentially catch too much though, I am not 100% up-to-date with the specification of a file URL.